src/gql/mocks/taskData.ts

```diff
@@ -17,7 +17,6 @@ export const taskQuery: TaskQuery = {
     details: {
       description:
         "Long description that requirese use of the inline definition component. This would include details about where the task failed.",
-      diskDevices: [],
```

Comment:
Whats going on here? I've seen this change in a few prs is there some merge conflict that needs to be resolved or maybe this branch needs to merge main?

Reply:
Sophie already fixed it on Friday but at the point where I committed it she hadn't fixed it yet